recipes/default.rb

```diff
@@ -46,6 +46,11 @@
   end
 end
 
+##  Set syncrepl_config dynamic values here
+node.default['openldap']['syncrepl_config']['searchbase'] = "\"#{node['openldap']['basedn']}\""
+node.default['openldap']['syncrepl_config']['binddn'] = "\"#{node['openldap']['syncrepl_cn']},#{node['openldap']['basedn']}\""
+node.default['openldap']['syncrepl_config']['credentials'] = "\"#{node['openldap']['slapd_replpw']}\""
```

Comment:
Hi Lamont, I would say it's a very useful use of attributes as it's functionally equivalent to the code it replaces. I put those three lines there as a way to maintain backwards compatibility rather than just replacing the attributes altogether, for example slapd_replpw with credentials. I could do as you suggest and pass these values in via template variables but my intent was not a major rewrite but just a simple code fix.